week-1/Homework/extra/2-piping.js

```diff
@@ -16,43 +16,49 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  //return parseFloat(num1) + parseFloat(num2).toFixed(1); //test failed
+  return (Number(num1) + Number(num2)).toFixed(1); //test failed
```

Comment:
Do you know why the test failed? Also, it's really good to see that you haven't assumed that the user will pass in numbers as inputs to this function. This is very good software engineering thinking! 💯

Reply:
You could also look at [parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) and [parseFloat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat). They do something similar!